app/main.py

```diff
@@ -1,3 +1,58 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: (int, float)) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: (int, float, Distance)) -> Distance:
+        return Distance(self.km + other.km) if \
+            isinstance(other, Distance) else \
+            Distance(self.km + Distance(other).km)
+
+    def __iadd__(self, other: (int, float, Distance)) -> Distance:
+        self.km = self.km + other.km if \
+            isinstance(other, Distance) else \
+            self.km + other
+        return self
+
+    def __mul__(self, other: (int, float)) -> Distance:
+        return Distance(self.km * other)
+
+    def __truediv__(self, operand: (int, float)) -> Distance:
+        other = round((self.km / operand), 2)
+        return Distance(other)
+
+    def __lt__(self, other: (int, float, Distance)) -> bool:
+        return self.km < other.km if \
+            isinstance(other, Distance) else \
+            self.km < other
+
+    def __gt__(self, other: (int, float, Distance)) -> bool:
+        return self.km > other.km if \
+            isinstance(other, Distance) else \
+            self.km > other
+
+    def __ge__(self, other: (int, float, Distance)) -> bool:
+        return self.km >= other.km if \
+            isinstance(other, Distance) else \
+            self.km >= other
+
+    def __le__(self, other: (int, float, Distance)) -> bool:
+        return self.km <= other.km if \
+            isinstance(other, Distance) else \
+            self.km <= other
```

Comment:
try to rewrite these two methods in one line each. Use previously created magic methods
Hint: `<=` is the same as `not >`